helm/alfresco-content-services/values.yaml

```diff
@@ -11,6 +11,8 @@
 
 repository:
   replicaCount: 2
+  # MNT-21042: enforce a Recreate update for the repository deployment
```

Comment:
Not sure if you need a reference to MNT here

Reply:
Thanks @killerboot, I just removed that reference.